### ISSUE TITLE
[Ch 23][Notebook][Major] 본편 §📐 "random 보다 일관되게 빠르고 낮게 수렴" — 부록 실측은 정반대 (A 0.547 < B 0.598, negative transfer)

### ISSUE THREAD

**Issue by fluentmin** [yoon-gu/neuqes-101#184]
> **요약** — 본편 §📐 가 "*위키 사전학습 본체* 가 … **random 보다 일관되게 빠르고 낮게 수렴**" 이라고 단정하는데, 그 근거로 가리키는 부록의 실측은 **정반대**입니다 (A 사전학습 0.5470 < B random init 0.5980, 3회 동일 — negative transfer). 부록 본문은 이미 "역전 가능" 을 전제로 쓰여 있어, **본편과 부록이 서로 다른 결론**을 말하고 있습니다.

## 무엇이 문제인가

본편 §📐 "사전학습 효과가 loss 곡선에 어떻게 드러나나" 아래 문단:

> random baseline 은 *두 셋업 모두 같음* — 사전학습이 *학습 속도* 와 *수렴점* 에 영향. … *위키 사전학습 본체* 가 NSMC 도메인에서 *완벽한 성능* 을 내지는 못해도, **random 보다 일관되게 빠르고 낮게 수렴** (사전학습 없는 random init 과의 직접 비교는 부록 …)

부록을 Colab T4 에서 3회 돌린 실측:

| 셋업 | accuracy | F1 | AUC |
|---|---|---|---|
| A (ours + MLM 3 epoch) | **0.5470** | 0.5017 | 0.5678 |
| B (random init, 같은 hyperparams) | **0.5980** | 0.5786 | 0.6359 |
| delta (A − B) | **−0.0510** | −0.0770 | −0.0681 |

부록은 `torch.manual_seed(SEED)` 가 걸려 있어 **3회가 소수점 4자리까지 동일**했고, 셀이 스스로 `>> Negative transfer: random init outperforms MLM-pretrained body.` 를 출력합니다. 부록의 §8 도 이 결과를 전제로 메커니즘을 설명합니다. 본편의 단정만 반대 방향입니다.

같은 챕터 안에서 본편 §📐 는 "일관되게 낮게 수렴", 부록은 "random 이 더 높다" 를 말합니다. Ch 22 #162, Ch 21 #128·#129 와 같은 계열 — 산문이 실측을 앞질러 단정한 자리입니다.

## 왜 고쳐야 하나

이 문장은 학습자가 부록을 열기 전에 결론을 미리 심어 줍니다. 부록의 핵심 교육 메시지가 *얕은 일반 도메인 사전학습은 transfer 효과를 보장하지 않는다* 인데, 본편이 먼저 "일관되게 낫다" 고 말해 두면 부록 결과가 오류처럼 읽힙니다.

## 제안

실측대로 적고 부록으로 넘기는 방향으로 정리하기를 제안합니다. 사전학습이 *학습 속도와 수렴점에 영향을 준다* 는 일반론은 남기되, **이 셋업(2K paragraphs × 3 epoch, NSMC 짧은 구어체)** 에서는 random init 대비 우위가 보장되지 않으며 실측은 오히려 낮게 나온다는 사실을 한 줄로 밝히고, 그 메커니즘과 정량은 부록에서 다룬다고 이어 주면 본편과 부록이 같은 방향을 보게 됩니다. 표의 "본체에 일반 한국어 어휘·문맥 구조가 들어 있어 헤드가 NSMC 분류로 비교적 빠르게 적응" 같은 메모도 같은 기준으로 손봐야 합니다.

## 재현

```bash
FORCE=1 BRANCH=ch-23/review-2026-09 ./executed/run_via_cli.sh appendix_random_baseline
python3 -c "import json,re;nb=json.load(open('executed/appendix_random_baseline.ipynb'));\
t=''.join(''.join(o.get('text',[])) for c in nb['cells'] for o in c.get('outputs',[]) if o.get('output_type')=='stream');\
print(re.findall(r'\[[AB]\] eval accuracy.*|>> .*',re.sub(r'\x1b\[[0-9;]*m','',t)))"
```

기준 커밋 `6f957a7` (`ch-23/review-2026-09`) / Colab T4 2026-08-30 부록 3회 ok, 세 번 모두 동일값

전파: Notebook → WikiDocs → 원고 tex

Related to #47 (챕터 이슈), #162 (Ch 22 동일 계열), #128 #129 (Ch 21 부록 계열)

**Comment by fluentmin** [comment]
브랜치 `ch-23/review-2026-09` 에서 수정했습니다 — `55d5757`.

수정본을 Colab T4 에서 다시 돌려 실행본을 갱신했습니다 (`7efdb1b`, 본편 2분 14초 · 부록 1분 57초, 코드셀 에러 0, `source_sha256` 스탬프 일치). 실행본 마크다운에서 옛 문구가 사라지고 새 서술이 들어간 것까지 확인했습니다. WikiDocs 페이지와 원고 tex 는 변환기가 따로 생성하므로 손대지 않았습니다 — 변환 재실행이 필요합니다.
